it/src/main/scala/com/wavesplatform/it/Docker.scala

```diff
@@ -527,6 +527,68 @@ class Docker(suiteConfig: Config = empty, tag: String = "", enableProfiling: Boo
 
     log.debug(s"$label: $x")
   }
+
+  def runMigrationToolInsideContainer(node: DockerNode, nodeConfig: Config): DockerNode = {
+    val id = node.containerId
+    takeProfileSnapshot(node)
+    updateStartScript(node)
+    stopContainer(node)
+    saveProfile(node)
+    saveLog(node)
+    client.startContainer(id)
+    Thread.sleep(60000)
+    revertStartScript(node)
+    stopContainer(node)
+    client.startContainer(id)
+    node.nodeInfo = getNodeInfo(node.containerId, node.settings)
+    Await.result(
+      node.waitForStartup().flatMap(_ => connectToAll(node)),
+      3.minutes
+    )
+    node
+  }
+
+  private def revertStartScript(node: DockerNode): Unit = {
+    val id = node.containerId
+    log.debug("Check migration tool output and revert start script")
+    val cmd: Array[String] =
+      Array(
+        "sh",
+        "-c",
+        s"""less /opt/waves/migration-tool.log | grep -ir completed && cp /opt/waves/start-waves.sh.bk /opt/waves/start-waves.sh && chmod +x /opt/waves/start-waves.sh"""
+      )
+    val execCmd = client.execCreate(id, cmd).id()
+    client.execStart(execCmd)
+  }
+
+  private def updateStartScript(node: DockerNode): Unit = {
+    val id = node.containerId
+
+    log.debug("Make backup copy of /opt/waves/start-waves.sh")
+    val cpCmd: Array[String] =
+      Array(
+        "sh",
+        "-c",
+        s"""cp /opt/waves/start-waves.sh /opt/waves/start-waves.sh.bk"""
+      )
+    val execCpCmd = client.execCreate(id, cpCmd).id()
+    client.execStart(execCpCmd)
+
+    log.debug("Change script for migration tool launch")
+    val scriptCmd: Array[String] =
+      Array(
+        "sh",
+        "-c",
+        s"""rm /opt/waves/start-waves.sh && echo '#!/bin/bash' >> /opt/waves/start-waves.sh &&
+             |echo 'java ${renderProperties(asProperties(genesisOverride))} -cp /opt/waves/waves.jar com.wavesplatform.matcher.MigrationTool /opt/waves/template.conf cb > /opt/waves/migration-tool.log' >> /opt/waves/start-waves.sh &&
+             |echo 'while true; do echo 1; sleep 10; done' >> /opt/waves/start-waves.sh &&
+             |chmod +x /opt/waves/start-waves.sh
+           """.stripMargin
```

Comment:
We could replace the script at the end. With this change:
1. We doesn't need to wait 60 seconds above
2. We should check the status of the container in a loop until it closed